docs/ballot.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Ballot Operation
+id: ballot
+author: Davis Sawali
+---
+
+
+The `Ballot` operation allows delegates to cast one `Yay`, `Nay`, or `Pass` ballot on a selected proposal. Delegates are only able to cast their votes during the **Exploration period**.
+
+
+## Examples
+The `Ballot` operation is currently available in the Contract API, and can be used as such:
+```typescript
+const op = await Tezos.contract.ballot({
+  period: 1,
+  proposal: 'PROPOSAL_HASH',
+  ballot: 'BALLOT_VOTE_STRING'
+});
+
+await op.confirmation();
+```
+- `period` is the voting period of the current protocol
```

Comment:
The period can be found with the RPC endpoint `votes/current_period`, correct?
Should we abstract this step from the user? Instead of accepting a parameter named `period`, we can fetch the current period by calling the RPC when preparing the operation (in the operation-emitter.ts file)

Reply:
that's actually a good idea. should we still give the option to pass the period manually or do we want to abstract it completely?